lib/text/ttml_text_parser.js

```diff
@@ -933,6 +947,32 @@ shaka.text.TtmlTextParser = class {
 
     return (milliseconds / 1000) + seconds + (minutes * 60) + (hours * 3600);
   }
+
+  /**
+   *
+   * @param {?string} cellResolution
+   * @return {!Array.<number>}
+   * @private
+   */
+  static getCellResolution_(cellResolution) {
+    const {Cue} = shaka.text;
+
+    // If not specified, the number of columns and rows must be considered
+    // to be 32 and 15 respectively
```

Comment:
You've already defined a default in the Cue constructor.  Why duplicate it?

Reply:
agree, 
changed the logic, so now cue.cellResolution default value will be replaced only if ttp:cellResolution was presented in ttml